python/foglamp/services/south/server.py

```diff
@@ -180,15 +180,18 @@ def __init__(self):
                 # pollInterval is expressed in milliseconds
                 sleep_seconds = int(self._plugin_handle['pollInterval']['value']) / 1000.0
                 await asyncio.sleep(sleep_seconds)
-                # If successful, then set retry count back to 1, meaning that
-                # only in case of 3 successive failures, exit.
-                try_count = 1
```

Comment:
why this is not set to 1 now?! and the comment

Reply:
Because it is already set to 1 at start.

Reply:
Why would we want to stop it? The process will continue to run and we can switch the sensortag off. To stop the process altogether, we will need to terminate the process, like any other South plugin.